### PR TITLE
Oc 123

### PR DESCRIPTION
**Pull request by ZaraGlynn**

### CODE REVIEW

**Comment by ZaraGlynn** [comment]
@Dan-Wood @mehrdadgit

File: src/items/items.test.ts [hunk 1]
```diff
@@ -3,12 +3,12 @@ require('dotenv').config(
     DEBUG: true
   });
 
-console.log('env pass:', process.env.PGPASSWORD)
+console.log('env pass:', process.env.PGPASSWORD);
```
Comment (Dan-Wood) [inline src/items/items.test.ts]: ?

Comment (mehrdadgit) [reply src/items/items.test.ts]: good catch, this was my debugging code; please clean up @ZaraGlynn

**Comment by mehrdadgit** [comment]
Happy with this one @Dan-Wood ? About to merge...

**Pull request merged**